steps/testproject-upload/0.1.0/step.yml

```diff
@@ -0,0 +1,74 @@
+title: Upload a file to TestProject
+summary: Upload an IPA/APK file to TestProject
+description: Upload an IPA/APK file to TestProject
+website: https://github.com/sca-organization/bitrise-testproject-upload
+source_code_url: https://github.com/sca-organization/bitrise-testproject-upload
+support_url: https://github.com/sca-organization/bitrise-testproject-upload/issues
+published_at: 2022-03-24T12:45:55.639574+11:00
+source:
+  git: https://github.com/sca-organization/bitrise-testproject-upload.git
+  commit: 4b32da488eebd3a2ef77c3f5c9ac2a0f5e186c5c
+project_type_tags:
+- ios
+- android
+- macos
+type_tags:
+- test
+toolkit:
+  bash:
+    entry_file: step.sh
+deps:
+  brew:
+  - name: jq
+  apt_get:
+  - name: jq
+is_requires_admin_user: true
+is_always_run: false
+is_skippable: false
+run_if: ""
+inputs:
+- apk_ipa_filepath: $BITRISE_IPA_PATH
+  opts:
+    description: |
+      This usually gets generated by the the "Gradle Runner" or "XCode Archive & Export" step
+      The app file you want to upload to TestProject, usually $BITRISE\_APK\_PATH or $BITRISE\_IPA\_PATH.
+    is_expand: true
+    is_required: true
+    is_sensitive: false
+    summary: Path of the IPA or APK that you want to upload.
+    title: Path of the IPA or APK
+- opts:
+    description: ""
+    is_expand: true
+    is_required: false
+    is_sensitive: false
+    summary: Your TestProject job id
+    title: TestProject job id
+  testproject_job_id: $TESTPROJECT_JOB_ID
```

Comment:
Please use only default values that are expected to be set by an other Step (like `$BITRISE_IPA_PATH`), to avoid confusion.

Reply:
removed. thanks